core/domain/takeout_service.py

```diff
@@ -46,6 +47,32 @@ def get_models_which_should_be_exported():
             base_models.EXPORT_POLICY.CONTAINS_USER_DATA]
 
 
+def export_all_profiles_data_by_parent_user(parent_user_id):
+    """Exports selected models according to model defined export_data functions
+        for all profile users linked to the parent_user of the given id.
+
+    Args:
+        parent_user_id: str. The user_id of the user whose associated profile
+            users' data has being exported.
+
+    Returns:
+        list. List of dictionaries containing all user data in the following
+        format:
+        {
+            <MODEL_NAME>_data: <dict of data in format as specified by
+                                model export policy>
+        }. List might be empty if no profiles are associated with the user
+        corresponding to the given parent_user_id.
+    """
+    all_profiles_user_ids = (
+        user.user_id for user in
+        user_services.get_all_profiles_auth_details_by_parent_user_id(
+            parent_user_id)
+    )
+    return [export_data_for_user(user_id) for user_id in
```

Comment:
Break after '['; see code styling guidelines.

Reply:
This function has been removed now.